extensions/binding/org.eclipse.smarthome.binding.onewire/ESH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,130 @@
+		<label>Temperature</label>
+		<description>temperature value of this sensor</description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+		<config-description>
+			<parameter name="offset" type="text">
+				<label>Offset</label>
+				<description>Offset for channel (with unit)</description>
+				<default>0 °C</default>
+			</parameter>
+			<parameter name="resolution" type="text">
+				<label>Sensor resolution</label>
+				<options>
+					<option value="9">9 bit</option>
+					<option value="10">10 bit</option>
+					<option value="11">11 bit</option>
+					<option value="12">12 bit</option>
+				</options>
+				<default>10</default>
+				<limitToOptions>true</limitToOptions>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</channel-type>
+	<!-- Dewpoint Channel -->
+	<channel-type id="dewpoint">
+		<item-type>Number:Temperature</item-type>
+		<label>Dewpoint</label>
+		<description>dewpoint (calculated from temperature and relative humidity)</description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<!-- Relative Humidity Channel -->
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>relative humidity (0-100%)</description>
+		<state readOnly="true" pattern="%d %%" />
+	</channel-type>
+	<channel-type id="humidityconf">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<description>relative humidity (0-100%)</description>
+		<state readOnly="true" pattern="%d %%" />
+		<config-description>
+			<parameter name="humiditytype" type="text">
+				<label>Humidity Sensor-Type</label>
+				<limitToOptions>true</limitToOptions>
+				<options>
+					<option value="/humidity">HIH-3610</option>
+					<option value="/HIH4000/humidity">HIH-4000</option>
+					<option value="/HTM1735/humidity">HTM-1735</option>
+					<option value="/DATANAB/humidity">Datanab</option>
+				</options>
+				<default>/humidity</default>
+				<required>true</required>
+			</parameter>
+		</config-description>
+	</channel-type>
+	<!-- Absolute Humidity Channel -->
+	<channel-type id="absolutehumidity">
+		<item-type>Number:Density</item-type>
+		<label>Abs. Humidity</label>
+		<description>absolute humidity (calculated from temperature and relative humidity)</description>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<!-- Voltage Channel -->
+	<channel-type id="voltage">
+		<item-type>Number:Voltage</item-type>
+		<label>Voltage</label>
+		<description></description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<!-- Supply-Voltage Channel -->
+	<channel-type id="supplyvoltage">
+		<item-type>Number:Voltage</item-type>
+		<label>Supply Voltage</label>
+		<description></description>
+		<state readOnly="true" pattern="%.1f %unit%" />
+	</channel-type>
+	<!-- Light Channel -->
+	<channel-type id="light">
+		<item-type>Number:Illuminance</item-type>
+		<label>Lightness</label>
```

Comment:
I was thinking about that, too. However the german "Helligkeit" which is used in the documentation of the manufacturer is translated to "lightness" according to dict.leo.org. iButtonLink calls it "light level" sensor. I can change it to whatever you think fits best.

Reply:
Since this seems to be a physical measurement value I suggest to use "Illuminance" or may be "Luminosity" (see https://dict.leo.org/englisch-deutsch/Lichtst%C3%A4rke). Is it measured in candela here? You may also provide a state description pattern.

Reply:
I changed it to illuminance since the measured value is in lux. State pattern added.